include/triton/core/tritonserver.h

```diff
@@ -785,8 +785,8 @@ TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_InferenceRequestFlags(
 TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_InferenceRequestSetFlags(
     TRITONSERVER_InferenceRequest* inference_request, uint32_t flags);
 
-/// Get the correlation ID of the inference request. Default is 0,
-/// which indictes that the request has no correlation ID. The
+/// Get the correlation ID of the inference request as an unsigned int. 
```

Comment:
"uinsigned integer"

What if the correlation ID is actually a string. You should document that an error is returned in that case.